Makefile

```diff
@@ -232,4 +232,4 @@ fmtcheck:
 
 .PHONY: validate_commits
 validate_commits: $(GOPATH)/bin/git-validation
-	git-validation -q -run short-subject,message_regexp='(Issue #[0-9]+ .*|cut v[0-9]+.[0-9]+.[0-9]+)' -range $(START_COMMIT_MESSAGE_VALIDATION)...
+	git-validation -q -run short-subject,message_regexp="^(Issue #[0-9]+ .*|cut v[0-9]+.[0-9]+.[0-9]+)$$" -range $(START_COMMIT_MESSAGE_VALIDATION)...
```

Comment:
Why was this changed to a double-quotes from single-quotes? This way you will have an issue with shell interpretation/evaluation of eg. variables (`$`)